source/FreeRTOS_ARP.c

```diff
@@ -141,6 +129,17 @@ eFrameProcessingResult_t eARPProcessPacket( ARPPacket_t * const pxARPFrame )
     const void * pvCopySource;
     void * pvCopyDest;
 
+    /* Next defensive request must not be sent for arpIP_CLASH_RESET_TIMEOUT_MS
+     * period. */
+    static TickType_t uxARPClashTimeoutPeriod = pdMS_TO_TICKS( arpIP_CLASH_RESET_TIMEOUT_MS );
+
+    /* This local variable is used to keep track of number of ARP requests sent and
+     * also to limit the requests to arpIP_CLASH_MAX_RETRIES per arpIP_CLASH_RESET_TIMEOUT_MS
+     * period. */
+    static UBaseType_t uxARPClashCounter = 0U;
```

Comment:
Does this work for multiple interfaces?

Reply:
it shouldn't work without synchronization from the interface